objects/namespace-install.yaml

```diff
@@ -2255,7 +2255,7 @@ spec:
         name: argocd-application-controller
         resources:
           limits:
-            memory: 2Gi
```

Comment:
can you add a memory request for 4Gi as well?

Reply:
the commit message also says `Incrase`

Reply:
request defaults to limits if unspecified and limit is explicitly specified

Reply:
We've had this conversation before haven't we... Just fix the commit message :-)